resources/validations/src/ssp.sch

```diff
@@ -1139,108 +1145,99 @@ A FedRAMP SSP must incorporate a procedure document for each of the 17 NIST SP 8
                         id="inventory-item-has-one-scan-type"
                         role="error"
                         test="not(oscal:prop[@name = 'scan-type'][2])">An inventory-item has only one scan-type property.</sch:assert>
-        </sch:rule>
-        <sch:rule context="oscal:inventory-item[oscal:prop[@name = 'asset-type' and @value = ('os', 'infrastructure')]]">
+            <!-- restrict the following to "infrastructure" -->
+            <sch:let name="is-infrastructure"
+                     value="exists(oscal:prop[@name = 'asset-type' and @value = ('os', 'infrastructure')])" />
             <sch:assert diagnostics="inventory-item-has-allows-authenticated-scan-diagnostic"
                         id="inventory-item-has-allows-authenticated-scan"
                         role="error"
-                        test="oscal:prop[@name = 'allows-authenticated-scan']">"infrastructure" inventory-item has
+                        test="not($is-infrastructure) or oscal:prop[@name = 'allows-authenticated-scan']">"infrastructure" inventory-item has
                         allows-authenticated-scan.</sch:assert>
             <sch:assert diagnostics="inventory-item-has-one-allows-authenticated-scan-diagnostic"
                         id="inventory-item-has-one-allows-authenticated-scan"
                         role="error"
-                        test="not(oscal:prop[@name = 'allows-authenticated-scan'][2])">An inventory-item has one-allows-authenticated-scan
-                        property.</sch:assert>
+                        test="not($is-infrastructure) or not(oscal:prop[@name = 'allows-authenticated-scan'][2])">An inventory-item has
+                        one-allows-authenticated-scan property.</sch:assert>
```

Comment:
B:Is there something particular you read in the guides that suggests infrastructure cannot have authenticated scans? Is it web only by implication? (I have notions about this, but I will double check myself.)

Reply:
FedRAMP OSCAL SSP guide page 56 "Mandatory for OS/Infrastructure. Leave blank for Software and Database."

Because of an arcane XSpec bug the test is a NOR: i.e., if it is infrastructure the the property is required.